learning_loop_node/detector/detector_node.py

```diff
@@ -434,47 +369,126 @@ async def sync_status_with_learning_loop(self) -> None:
 
         self.log_status_on_change(status.state or 'None', status)
```

Comment:
According to the typing, status.state must be NodeState, which should never result in "False"

```suggestion
        self.log_status_on_change(status.state, status)
```